src/styles/blocks/card.scss

```diff
@@ -0,0 +1,17 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
let's set this property only one time global

Reply:
which of them? border-box or display flex?